frontend/src/utils/RequestHeaders.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Utility class for various request Headers.
+ */
+
+export const createRequestHeader = () => ({
```

Comment:
Okay, I'll check it out. Are we ever going to need different headers at any point?

Reply:
None that I can think of. Coz the backend should always remain the same for frontend. Honestly the only two different headers I can think of is the localhost:3000/env call vs every call that goes to API_URL. which is not worth the effort of implementing axios instances.

If we do end up in a situation where there is multiple headers for different endpoints it might be good to implement :)